Makefile

```diff
@@ -199,11 +199,9 @@ endif
 
 .PHONY: cmultibuildpush
 cmultibuildpush: ## Build and push multi arch container image
-ifndef CONTAINER_TOOL
-	$(error No container tool (docker, podman) found in your environment. Please, install one)
+ifndef DOCKER_CMD
+	$(error Presently, only docker is supported for multi architecture image build)
```

Comment:
Also mention that docker wasn't detected. Also add a comment to instruct them to install docker and try again

Reply:
"Docker wasn't detected. Please install docker and try again."